A3A/addons/ultimate/config/cfgForbiddenItemsGM.hpp

```diff
@@ -2,4 +2,4 @@ class GM_limited_base : forbidden_limited_base {
 	addons[] = {"gm_core"};
 };
 
-class gm_p2a1_launcher_blk : GM_limited_base {};
\ No newline at end of file
+class gm_p2a1_launcher_blk : GM_limited_base {}; ///why?
```

Comment:
probably a disposable launcher, idk, I didn't make this one

Reply:
I'll try commenting it out to see what happens.

Reply:
I doubt you'll figure out why it's there, it'll likely end up being discovered 3 months later that it broke something obscure